x-pack/winlogbeat/module/security/config/winlogbeat-security.js

```diff
@@ -1030,6 +1046,15 @@ var security = (function () {
         return msobjsMessageTable[code];
     };
 
+    var addActionDesc = function(evt){
+        var code = evt.Get("event.code");
+        if (!code) {
+            return;
+        }
+        var eventActionDescription=eventActionTypes[code];
+        evt.Put("event.action",eventActionDescription)
```

Comment:
```suggestion
        evt.Put("event.action", eventActionDescription);
```

Reply:
To be defensive I would check that a valid action was returned from the event ID lookup. Like

```
if (eventActionDescription) {
    evt.Put("event.action", eventActionDescription);
}
```